src/GSM3MobileMockupProvider.cpp

```diff
@@ -114,7 +114,11 @@ int GSM3MobileMockupProvider::readSMS()
 int GSM3MobileMockupProvider::connectTCPClient(const char* server, int port, int id_socket)
 {
 	Serial.println("GSM3MobileMockupProvider::connectTCPClient()");
-	Serial.print(server);Serial.print(":");Serial.print(port);Serial.print("-");Serial.println(id_socket);
+	Serial.print(server);
+	Serial.print(':');
+	Serial.print(port);
+	Serial.print('-');
+	Serial.println(id_socket);

```

Comment:
I broke on multiple lines the coded, it looked hard to read on one line in my opinion. However, if this edit is unwanted, give me a sec and I'll fix the commit of this PR reverting this line breakage.